### DISCUSSION TITLE
add way of excluding attributes from display/search attributes when using a wildcard

### DISCUSSION THREAD

**Discussion by dvc94ch** [meilisearch/product#672] [Feedback & Feature Proposal]
I took a brief look at the code, unless I'm missing something it doesn't seem to be possible

**Comment by macraig** [comment]
Hi @dvc94ch, thanks for taking the time to give us feedback on the product! I want to make sure I understand your use case. Instead of adding specific attributes to `displayedAttributes` and `searchableAttributes`, you'd like to have a way of adding all attributes (using the wildcard) excluding a few of them. Is that accurate?

**Reply by dvc94ch** [reply]
Yes, that's correct. Because I don't know the schema of the data in advance, only the primary key and some special keys, but the rest is provided by the user as metadata, a list of key/value pairs

**Reply by dvc94ch** [reply]
is it just my project or is this a more generally useful feature?

**Reply by macraig** [reply]
Absolutely, your feature request does seem interesting and could potentially provide value to other users. Hence, we're going to leave this discussion open so that others who might be experiencing similar challenges can contribute :)

However, we want to be transparent about our current priorities. Our team is presently occupied with other pressing commitments and projects, so we may not be able to address this specific request right away. That being said, we genuinely appreciate your suggestion and we're always looking for ways to improve our offerings based on user feedback.

We hope you understand our position and we'll certainly keep your idea in mind for the future. Please feel free to continue to provide input and share your thoughts - your input is extremely valuable to us.

**Reply by iigorr** [reply]
any development on excluding attriubtes?

**Reply by curquiza** [reply]
Hey, it's logged on our side, but not prioritized yet due to our small engineering bandwidth
